docs/guides/debugging/clion.md

```diff
@@ -0,0 +1,51 @@
+# Debugging in CLion
+
+In this guide, we’ll be setting up IntelliJ CLion for debugging the [Core Process of your Tauri app].
+
+## Prerequisites
+
+You need to install the [IntelliJ Rust Plugin](https://plugins.jetbrains.com/plugin/8182-rust/docs) that Rust features are enabled.
```

Comment:
> that Rust features are enabled.

Something is missing and i have no idea what 😅

Reply:
yeah fixed it!